README.md

```diff
@@ -164,6 +164,45 @@ add-apt-repository ppa:rpi-distro/ppa
 apt-get install -y pigpio
 ```
 
+### Custom post-build script
+
+You may want to perform arbitrary post-processing on your build outputs, in the event of a sucessful build - use `--custom-post-build-script` for this.
+
+Keep in mind
+It is run at the root of the built workspace.
+
+Following is an example setup that allows a user to run [colcon bundle](https://github.com/colcon/colcon-bundle) to create a portable bundle of the cross-compiled application.
+
+Here are the contents of `./postbuild.sh`
+
+```bash
+#!/bin/bash
+set -eux
+
+apt-get update
+apt-get install -y wget
+wget http://packages.osrfoundation.org/gazebo.key -O - | apt-key add -
+
+apt-get install -y python3-apt
+pip3 install -u setuptools pip
+pip3 install -U colcon-ros-bundle
+
+colcon bundle \
+  --build-base build_"${TARGET_ARCH}" \
```

Comment:
Nit: does `TARGET_ARCH` need to be exported in the script?

Reply:
It is defined in the calling environment. The 'set -u' flag at the top will make it fail with an error if the variable is not set.